topics/Dialog-and-popup-sizes.md

```diff
@@ -0,0 +1,278 @@
+<!-- Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license. -->
+
+# Dialog and Popup Sizes
+
+<link-summary>Guidelines on choosing the correct size when creating a dialog or a popup</link-summary>
+
+Follow these guidelines to select the correct size when creating a dialog or a popup for IntelliJ-based products.
+
+## Default sizes
+
+### Dialog
+
+There are four recommended window sizes for dialogs: **extra small**, **small**, **medium**, and **large**. Select the size depending on the amount of content to keep the important information visible.
+
+<note>When setting the default size for your dialog, it should be impossible to make it smaller.</note>
+
+#### Extra small
+
+Size: 350×250 px.
+
+Best for: several components that are stacked vertically.
+
+For example, the <control>Rename File</control> dialog has several input fields and checkboxes in one column:
+
+![The Rename dialog with two input fields, two checkboxes, and actions in the dialog footer. The components are stacked vertically in one column](dialog_popup_sizes_extra_small.png){width="706"}
+
+#### Small
+
+Size: 500×350 px
+
+Best for:
+* Two-column layout with a tree on one column and several components that are stacked vertically in the second column.
+* Multiple components with short labels that are divided into two columns.
+* A full-width table with two or three columns.
+* A full-width code editor or snippet.
+
+For example, the <control>Evaluate Expressions</control> dialog has two code snippets that are stacked vertically:
+
+![The Evaluate Expressions dialog that has two code snippets stacked vertically](dialog_popup_sizes_small.png){width="706"}
+
+#### Medium
+
+Size: 750×525 px
+
+Best for:
+* A full-width table with four or more columns.
+* Two-column layout with an image or a video in any of the columns. In this case, the image or the video takes the most space.
+* Three-column layout with a code editor or snippet in any of the columns.
+* Two code editors or snippets in a row.
+
+For example, the <control>Image Viewer</control> dialog has a two-column layout with image or video in the first column and the list of input fields with actions in the second column:
+
+![The Recent Files popup that has a two-column layout. There are quick links in the first column and the list of recent files with long names in the second column](dialog_popup_sizes_medium.png){width="706"}
```

Comment:
It's the old alt text about **recent files** popup. Requires an updated version of the **image viewer** dialog